README.md

```diff
@@ -41,14 +41,18 @@ principal charm.
 See [config.yaml](config.yaml) for
 list of configuration options.
 
+> Note: Setting HTTP proxy values will be overriden if `juju-http-proxy` or `juju-https-proxy` settings are set on the model.
+
+# Testing
+
+When upgrading this charm, the [`container_runtime_version`](https://github.com/VariableDeclared/jenkins/blob/3bf2cabc7185568e6a80a137a7f083c428048683/jobs/validate-juju-https-envs.yaml#L25) of the `validate-https-proxy-envs` key needs to be updated with the new containerd version.
```

Comment:
If I'm understanding correctly, following the trail here:
* [validate-juju-http.yaml:25](https://github.com/charmed-kubernetes/jenkins/blob/4d2d33582cd16e8311fb7bbe9f496dbe94b23258/jobs/validate-juju-http.yaml#L25)
* [validate-juju-http-proxy/Jenkinsfile:44](https://github.com/charmed-kubernetes/jenkins/blob/4d2d33582cd16e8311fb7bbe9f496dbe94b23258/jobs/validate-juju-http-proxy/Jenkinsfile#L44)
* [integration/test_juju_https_proxy.py:88-91](https://github.com/charmed-kubernetes/jenkins/blob/4d2d33582cd16e8311fb7bbe9f496dbe94b23258/jobs/integration/test_juju_https_proxy.py#L88-L91)

The `container_runtime_version` is a hard-coded revision of the charm in the charm store, correct?

That's generally not something we want, because we want to be testing regularly against the latest stable and latest edge. That won't happen if the charm revision is hard-coded.

I suggest removing this text from the README and following up on the test later to ensure we don't have to bump the revision regularly.

Reply:
OK!